### PR TITLE
Add task to create a release candidate.

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

More release automation.

### Proposed Changes

Task to create a release candidate from develop and push it to github.

### Reason for Changes

Automation.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -800,6 +800,18 @@ function getRebuildBranchName() {
...
+// Helper function: get a name for a rebuild branch. Format: rebuild_month_yyyy.
+function getRCBranchName() {
+  var date = new Date();
+  var monthNames = [
+    'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+    'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+  ];
+  var month = monthNames[date.getMonth()];
+  var yy = date.getFullYear().slice(-2);
+  return 'rc_' + month + '_' + yyyy;
```
Comment (alschmiedt) [inline gulpfile.js]: This is different from our current naming scheme so are we going to have to update the script that creates the release doc?

[the remark this answers is not in the document]
Comment (NeilFraser) [reply gulpfile.js]: Actually, I'd love it if we could change the format to drop the English months from the name.  The alphabetically-sorted list of branches in GitHub is useless at the moment.  Would vastly prefer YYYY-MM.

[the remark this answers is not in the document]
Comment (rachel-fenichel) [reply gulpfile.js]: Nope.  I was looking for opinions first.  And boy did I get them!

File: gulpfile.js [hunk 2]
```diff
@@ -800,6 +800,18 @@ function getRebuildBranchName() {
...
 
+// Helper function: get a name for a rebuild branch. Format: rebuild_month_yyyy.
+function getRCBranchName() {
+  var date = new Date();
+  var monthNames = [
+    'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+    'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+  ];
+  var month = monthNames[date.getMonth()];
+  var yy = date.getFullYear().slice(-2);
```
[the remark this answers is not in the document]
Comment (rachel-fenichel) [reply gulpfile.js]: If we're still using this the next time we get a Y2K bug, we've got other problems.

File: gulpfile.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (samelhusseini) [reply gulpfile.js]: Actually, there’s really no need to keep these branches around anyway, as we use tags that are well sorted with semver.

**Comment by rachel-fenichel** [comment]
Updated, and I tested by running `npx gulp git-create-rc` and verifying that a branch named rc_2019_10 showed up on github and had the right contents.

**Pull request merged**